frontend/views/containers/proposals/AddMembers.vue

```diff
@@ -139,10 +139,10 @@ export default {
           break
         }
       }
-
       if (!hasFailed) {
         this.ephemeral.currentStep += 1 // Show Success step!
       }
+      done()
```

Comment:
Same concerns as with the other callback function in Contributions.vue (see comment below)

Reply:
Sorry, I'm not following... why is this `done()` needed?

Reply:
it isn't anymore. Was the only one I forgot to remove.